.github/workflows/release_community_cromwell.yml

```diff
@@ -83,7 +89,7 @@ jobs:
       run: |
         set -e
         cd cromwell
-        git push origin master develop
+        git push origin develop master
```

Comment:
TIL you can push multiple branches to origin simultaneously. At first I thought this was somehow setting one to the other, but it is not so.